yarn-project/noir-protocol-circuits/src/crates/private-kernel-lib/src/common.nr

```diff
@@ -408,3 +423,7 @@ pub fn validate_call_against_request(private_call: PrivateCallData, request: Cal
         );
     }
 }
+
```

Comment:
I'd add a `TODO` to remind us to derive this properly eventually. The parameters will need to be the parent public key and the contract address.
I'm going to stop procrastinating and peeking at this exciting project and go back to reviewing the yp now.